e2e/test/scenarios/metrics/metrics-editing.cy.spec.ts

```diff
@@ -0,0 +1,527 @@
+      verifyScalarValue("9,380");
+    });
+
+    it.skip("should create a metric with a custom aggregation expression based on 2 metrics (metabase#42253)", () => {
+      createQuestion(ORDER_COUNT_DETAILS);
+      createQuestion(PRODUCT_COUNT_DETAILS);
+      startNewMetric();
+      popover().findByText("Metrics").click();
+      popover().findByText(ORDER_COUNT_DETAILS.name).click();
+      startNewJoin();
+      popover().within(() => {
+        cy.findByText("Sample Database").click();
+        cy.findByText("Raw Data").click();
+        cy.findByText("Metrics").click();
+        cy.findByText(PRODUCT_COUNT_DETAILS.name).click();
+      });
+      getNotebookStep("summarize").findByText(ORDER_COUNT_DETAILS.name).click();
+      enterCustomColumnDetails({
+        formula: `[${ORDER_COUNT_DETAILS.name}] / [${PRODUCT_COUNT_DETAILS.name}]`,
+        name: "",
+      });
+      popover().button("Update").click();
+      saveMetric();
+      runQuery();
+      // FIXME put correct value verifyScalarValue("9,380");
+    });
+
+    it("should add an aggregation clause in a metric query with 2 stages", () => {
+      startNewMetric();
+      popover().findByText("Raw Data").click();
+      popover().findByText("Orders").click();
+      addAggregation({ operatorName: "Count of rows" });
+      addBreakout({ columnName: "Created At", bucketName: "Year" });
+      addAggregation({
+        operatorName: "Count of rows",
+        isPostAggregation: true,
+      });
+      saveMetric();
+      runQuery();
+      verifyScalarValue("5");
+    });
+
+    it("should add multiple aggregation columns in the first stage of a metric query", () => {
+      startNewMetric();
+      popover().findByText("Raw Data").click();
+      popover().findByText("Orders").click();
+      addAggregation({ operatorName: "Sum of ...", columnName: "Total" });
+      addAggregation({ operatorName: "Sum of ...", columnName: "Subtotal" });
+      addBreakout({ columnName: "Created At" });
+      addAggregation({
+        operatorName: "Average of ...",
+        columnName: "Sum of Subtotal",
+        isPostAggregation: true,
+      });
+      saveMetric();
+      runQuery();
+      verifyScalarValue("29,554.86");
+    });
+
+    it.skip("should combine multiple aggregation columns in a custom aggregation expression (metabase#42425)", () => {
+      startNewMetric();
+      popover().findByText("Raw Data").click();
+      popover().findByText("Orders").click();
+      addAggregation({ operatorName: "Sum of ...", columnName: "Total" });
+      addAggregation({ operatorName: "Sum of ...", columnName: "Subtotal" });
+      addBreakout({ columnName: "Product ID" });
+      startNewAggregation({ isPostAggregation: true });
+      popover().findByText("Custom Expression").click();
+      enterCustomColumnDetails({
+        formula: "[Sum of Total] - [Sum of Subtotal]",
+        name: "Metric",
+      });
+      popover().button("Done").click();
+      saveMetric();
+      runQuery();
```

Comment:
Missing assertion. We can leave a "FIXME" comment like in the previous skipped test.